### ISSUE TITLE
Authentication Uplift: review accessibility requirements

### ISSUE THREAD

**Issue by markverstege** [ConsumerDataStandardsAustralia/standards-experimental#24]
# Description
**Consider WCAG accessibility requirements**
Different channels and modalities used in an authentication process may be challenging for people with additional needs.
Some people with cognitive disabilities cannot solve puzzles, memorise a username and password, or retype one-time passcodes. The purpose of this ticket is to outline WCAG requirements specific to authentication and app switching and determine the best course of action to make CDR authentication accessible and inclusive to everyone. Noting other existing accessibility requirements apply, for this particular instance, the following success criteria have been identified and analysed as relevant:

1. SC 3.3.8  and 3.3.9 Accessible Authentication (AA and AAA): Applies when cognitive function tests that are used to authenticate users. Examples include remembering a password or solving a puzzle.
2. SC 1.3.5 Input Purpose and 4.1.2 Name, Role, Value: A web site uses a properly marked up username (or email) and password fields as the login authentication
3. SC 2.2.1 Timing Adjustable (Level A) and 2.2.6 Timeouts (Level AAA): Provide users with enough time to complete tasks. If there's a timeout during the login process, users must have the ability to extend the time or be warned before the timeout occurs. If a timeout results in data loss or requiring re-authentication, users must be warned about this and given an option to extend the session.
4. SC 3.2.5 Change on Request (Level AAA): This Success Criterion aims to eliminate potential confusion that may be caused by unexpected changes of context such as automatic launching of new windows

# Intention and Value of Change
Ensure accessibility for consumers is appropriately implemented and that it is achievable to implement

# Area Affected
CX Standards - Authentication

# Change Proposed
Refer https://miro.com/app/board/uXjVKJKxWNs=/?moveToWidget=3458764595382015108&cot=14
